tests/src/python/test_provider_postgres.py

```diff
@@ -429,6 +429,89 @@ def testPktIntInsert(self):
         self.assertNotEqual(f[0]['pk'], NULL, f[0].attributes())
         vl.deleteFeatures([f[0].id()])
 
+    def testPktUpdateBigintPk(self):
+        """Test if we can update objects with positive, zero and negative bigint PKs."""
+        vl = QgsVectorLayer('{} sslmode=disable srid=4326 key="pk" table="qgis_test".{} (geom)'.format(self.dbconn, 'bigint_pk'), "bigint_pk", "postgres")
+        dp = vl.dataProvider()
+        flds = dp.fields()
```

Comment:
```suggestion
        flds = vl.fields()
```

In this particular case it's completely equal to what you proposed, but let's use best practice in our code base.